windows-driver-docs-pr/install/inf-addservice-directive.md

```diff
@@ -244,6 +245,14 @@ For information about security descriptor strings, see [Security Descriptor Defi
 
 For more information about how to specify security descriptors, see [Creating Secure Device Installations](creating-secure-device-installations.md).
 
+**RequiredPrivileges**=*privilege-name*[**,**_privilege-name_]...
+> [!NOTE]
+> This value can only be used for *Win32 Services* and is only available on Windows 7 and above.
+
+Each *privilege-name* in the list is the name of the privilege.  See [Privilege Constants (Windows)](/windows/win32/secauthz/privilege-constants) for a list of the privilegs names. For each privilege name, only the text name is required, *For Example* The privilege name should be written as "SetAuditPrivilege" but not SE_AUDIT_NAME.
```

Comment:
why "*For Example* The" (i.e. why is "For Example" italicized and why is "Example" and "The" capitalized?  Were you modeling it off of somewhere else on the page?  I would probably just do "For example, the" (note also the addition of the comma)

Reply:
oh, and I would turn the comma at the end of "required," into a period.

Reply:
Sure, will take care of it